### PR TITLE
Fix vsprintf error

### PR DESCRIPTION
**Pull request by WouterFlorijn**
## This PR closes/fixes the following issues:
[//]: # (If for some reason your pull request does not require a test case you can just mark this box as checked and explain why it does not require a test case.)
 - Closes bunq/sdk_php#229
    - [x] Tested

### CODE REVIEW

File: src/Http/Handler/ResponseHandlerSignature.php [hunk 1]
```diff
@@ -168,7 +168,7 @@ private function ensureHeaderIsCorrectlyCased(string $headerName)
         if ($regexResult != self::REGEX_CHECK_FAILED) {
             foreach ($matches[self::INDEX_FIRST] as $match) {
                 $matchUpper = strtoupper($match);
-                $headerName = preg_replace(vsprintf(self::REGEX_REPLACE, $match), $matchUpper, $headerName);
+                $headerName = preg_replace(vsprintf(self::REGEX_REPLACE, [$match]), $matchUpper, $headerName);
```
Comment (MathijsvanVelde) [inline src/Http/Handler/ResponseHandlerSignature.php:171]: This does the same
```suggestion
                $headerName = preg_replace(sprintf(self::REGEX_REPLACE, $match), $matchUpper, $headerName);
```

Comment (WouterFlorijn) [reply src/Http/Handler/ResponseHandlerSignature.php:171]: @MathijsvanVelde yes, do you want me to update the PR with that suggestion?

**Review by timvisee — APPROVED** [review]

**Review by haddocc — APPROVED** [review]

**Comment by ralphjsmit** [comment]
@andrederoos @angelomelonas @bunq-bot Sorry to be tagging you, but you have previous merged pull requests in this repository. It would be much appreciated if you could quickly merge and release this urgent bug - **it makes the Bunq PHP SDK unusable for everyone**. Thanks!

**Comment by WouterFlorijn** [comment]
@angelomelonas do you know who that would be? Doesn't look like there are any recently active contributors here.

File: src/Http/Handler/ResponseHandlerSignature.php
Comment (sailingdeveloper) [inline src/Http/Handler/ResponseHandlerSignature.php:1]: Can you instead leave it vsprintf, but add the array brackets? I believe that’s more consistent with the rest of the code.

**Review by sailingdeveloper — CHANGES_REQUESTED** [review]
Can you instead leave it vsprintf, but add the array brackets? I believe that’s more consistent with the rest of the code.

**Comment by sailingdeveloper** [comment]
Additionally, I think the code here is auto-generated, so we might need to fix this on our internal side, or this change will be overridden in the next update.

**Comment by angelomelonas** [comment]
@WouterFlorijn I see @sailingdeveloper is here to help :)

**Comment by WouterFlorijn** [comment]
@sailingdeveloper Alright, updated!

**Pull request merged**